src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyApplicant1GenericTemplateHandler.java

```diff
@@ -68,8 +68,8 @@ public Map<String, String> addProperties(CaseData caseData) {
         );
     }
 
-    private String getNotificationTemplate() {
-        return notificationsProperties.getNotifyLipUpdateTemplate();
+    private String getNotificationTemplate(CaseData caseData) {
+        return caseData.isBilingual() ? notificationsProperties.getNotifyApplicantForHwfFeePaymentOutcomeInBilingual() : notificationsProperties.getNotifyLipUpdateTemplate();
```

Comment:
is bilingual notification only for HWF or pay by card , by assumption it should be only  for hwf hearing

Reply:
@jeswanth-hmcts As discussed with you and @dharmendrak We will be sending bilingual email for both gov pay and fee payment outcome event. So this is fine